### ISSUE TITLE
Animated RN 0.62

### ISSUE THREAD

**Issue by Thomazella** [jshanson7/react-native-swipeable#96]
`, { useNativeDriver: true }` second arg must be passed to Animated.event to silence a warning.

**Comment by parnekov** [comment]
@Thomazella Where file is located? I mean where should be added { useNativeDriver: true } to Animated.event?
Thanks.

**Comment by Thomazella** [comment]
Second argument to Animated.event function. { useNativeDrive: boolean}. I think every function call requires this now.

**Comment by parnekov** [comment]
@Thomazella Thank you. But my app crashes after put second argument. Without second argument works fine :)

**Comment by VSV6** [comment]
@Thomazella, Yes where I need to apply this code `{ useNativeDriver: true }.` And I'm not using any animation except this lib. That's it. Can you please be more specific? Thanks!

**Comment by rthomazel** [comment]
Animated.Event second argument I believe. Read the types it helps. I've been away from react native development for a few months so I could be outdated
